databases/orientdb/src/main/java/com/gentics/mesh/graphdb/cluster/TopologyEventBridge.java

```diff
@@ -136,6 +136,7 @@ public void onDatabaseChangeStatus(String nodeName, String iDatabaseName, DB_STA
 			statusInfo.put("node", nodeName);
 			statusInfo.put("database", iDatabaseName);
 			statusInfo.put("status", iNewStatus.name());
+			statusInfo.put("online", iNewStatus == DB_STATUS.ONLINE);
```

Comment:
Here we get rid of manual parsing of the OrientDB node status, which happens in the Core below.